### ISSUE TITLE
HTTP Loopback Connections are not enabled on this server

### ISSUE THREAD

**Issue by Isaac23164** [ThemeFuse/Unyson#4221]
Hi

I am using a theme based on Unyson and in backup tool I get this message:

`Important: HTTP Loopback Connections are not enabled on this server. If you need to contact your web host, tell them that when PHP tries to connect back to the site at the URL `https://mysite.com/wp-admin/admin-ajax.php` and it gets the error `Response code: 403`. There may be a problem with the server configuration (eg local DNS problems, mod_security, etc) preventing connections from working properly.`

The hosting company "hostinger" does not know the reason. Any advice ?

Thanks

**Comment by klynicol** [comment]
Yes, getting the same thing from Dreamhost. They don't allow calling php files from php files. Can someone please tell me why this is necessary in the Wordpress plugin? Why not just include the files you need instead?

**Comment by wn-waqar** [comment]
Important: HTTP Loopback Connections are not enabled on this server. If you need to contact your web host, tell them that when PHP tries to connect back to the site at the URL `https://americanmediamax.com/wp-admin/admin-ajax.php` and it gets the error `Response code: 500`. There may be a problem with the server configuration (eg local DNS problems, mod_security, etc) preventing connections from working properly.

The hosting company "hostinger"
